snap/snapcraft.yaml

```diff
@@ -15,6 +15,7 @@ apps:
   wifi-connect: 
     command: cmd 
     plugs: [network, network-control, network-bind, network-manager, control]
+    aliases: [wifi-connect_app]
```

Comment:
perhaps I am not up to date. I thought aliases are being removed from snapcraft.yaml: https://forum.snapcraft.io/t/improving-the-aliases-implementation/18